app/src/main/java/com/duckduckgo/app/global/view/FireDialog.kt

```diff
@@ -93,12 +101,37 @@ class FireDialog(
             cancel()
         }
 
+        if (appBuildConfig.sdkInt == Build.VERSION_CODES.O) {
```

Comment:
I'd change this condition to 
`if (appBuildConfig.sdkInt >= Build.VERSION_CODES.O)` as there are still people who use the navigation bar on O+.

Reply:
On APi 27+ the system takes care of it for us, please see the three button demo videos e.g. API 27

Reply:
I tested on a real device, Pixel 4a, Android 12 (API 32) and the navigation bar doesn't look great. I'll link a video in the task.